Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:1.9.3-p551
+ARG BUNDLER_VERSION=1.15.4
+
+RUN apt-get update -qq && \
+    apt-get install -qy curl jq python3 python3-dev python3-pip python3-yaml && \
+    pip3 install awscli && \
+    apt-get clean
+
+RUN useradd -c 'builder of ruby projs' -m -d /home/builder -s /bin/bash builder
+
+ENV GEM_HOME /usr/local/bundle
+RUN chmod -R 777 "$GEM_HOME"
```

Comment:
why 777? and is this only to build the gem?

Reply:
following @careyrp's example here 😄

Reply:
This a change I pulled from a newer version of the ruby docker image. Without it, only admin can install gems.
https://github.com/docker-library/ruby/blob/master/2.6/alpine3.9/Dockerfile#L129